Dockerfile

```diff
@@ -2,18 +2,18 @@
 #
 # BUILD: docker build --no-cache --rm -t nordaaker/convos .
 # RUN:   docker run -it --rm -p 8080:3000 -v /var/convos/data:/data nordaaker/convos
-FROM alpine:3.4
+FROM alpine:3.5
 MAINTAINER jhthorsen@cpan.org
 
-WORKDIR /
+ARG port=3000
 
-RUN apk update && \
-  apk del wget && \
-  apk add perl perl-io-socket-ssl perl-dev g++ make openssl openssl-dev wget curl && \
-  curl -L https://github.com/Nordaaker/convos/archive/stable.tar.gz | tar xvz && \
+RUN \
+  apk add -U perl perl-io-socket-ssl && \
+  apk add -t builddeps build-base perl-dev wget && \
+  wget -q -O - https://github.com/Nordaaker/convos/archive/stable.tar.gz | tar xvz && \
   /convos-stable/script/convos install && \
-  apk del perl-dev g++ make openssl openssl-dev curl && \
-  rm -rf /root/.cpanm /usr/local/share/man/*
```

Comment:
I can confirm that the man directory is already empty. Alpine removes manpages by default.